training_project/pages/reset_password_page.py

```diff
@@ -0,0 +1,30 @@
+from selenium.webdriver.common.by import By
+
+from training_project.pages.base_page import BasePage
+
+
+class ResetPasswordPage(BasePage):
+    USERNAME_INPUT = (By.XPATH, '//*[@id="app"]/div[1]/div[1]/div/form/div[1]/div/div[2]/input')
+    RESET_PASSWORD_FORM = (By.XPATH, '//*[@id="app"]/div[1]/div[1]/div')
+    CANCEL_BUTTON = (By.XPATH, '//*[@id="app"]/div[1]/div[1]/div/form/div[2]/button[1]')
+    RESET_PASSWORD_BUTTON = (By.XPATH, '//*[@id="app"]/div[1]/div[1]/div/form/div[2]/button[2]')
+
+    reset_password_link = "https://opensource-demo.orangehrmlive.com/web/index.php/auth/sendPasswordReset"
```

Comment:
reset url - https://opensource-demo.orangehrmlive.com/web/index.php/auth/sendPasswordReset
login url - https://opensource-demo.orangehrmlive.com/web/index.php/auth/login
I would say 'https://opensource-demo.orangehrmlive.com' is base url and other url we get by combining base and some endpoints
resetendpoint - web/index.php/auth/sendPasswordReset
loginendpoint - web/index.php/auth/login